staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/types.go

```diff
@@ -61,18 +61,18 @@ type ExecCredentialStatus struct {
 	// +optional
 	ExpirationTimestamp *metav1.Time `json:"expirationTimestamp,omitempty"`
 	// Token is a bearer token used by the client for request authentication.
-	Token string `json:"token,omitempty"`
+	Token string `json:"token,omitempty" datapolicy:"token"`
 	// PEM-encoded client TLS certificates (including intermediates, if any).
-	ClientCertificateData string `json:"clientCertificateData,omitempty"`
+	ClientCertificateData string `json:"clientCertificateData,omitempty" datapolicy:"security-key"`
```

Comment:
maybe I missed anything in the disucssion? This is not critical, only the key data is, right?